src/Files/Helpers/BitmapHelper.cs

```diff
@@ -19,5 +22,31 @@ public static async Task<BitmapImage> ToBitmapAsync(this byte[] data)
             await image.SetSourceAsync(ms.AsRandomAccessStream());
             return image;
         }
+
+
+        /// <summary>
+        /// Rotates the image at the specified file path.
+        /// </summary>
+        /// <param name="filePath">The file path to the image.</param>
+        /// <param name="rotation">The rotation direction.</param>
+        public static async Task Rotate(string filePath, BitmapRotation rotation)
+        {
+            if (string.IsNullOrEmpty(filePath))
+            {
+                return;
+            }
+
+            var file = await StorageFile.GetFileFromPathAsync(filePath);
```

Comment:
Use `StorageHelpers.ToStorageItem<StorageFile>()` and check if it's not null

Reply:
Also, this won't work with remote storage (phone storage etc.)
You'd have to detect if the function fails, and use fulltrust process

Calling @gave92 for help

Reply:
@d2dyno1 When I use the StorageHelpers.ToStorageItem<StorageFile>() method I get a InvalidCastException? from the ToStorageItem method. 

![image](https://user-images.githubusercontent.com/79826944/150660567-d1ba9087-c0c6-494e-9e10-c2565d637d6a.png)

![image](https://user-images.githubusercontent.com/79826944/150660539-41ef0228-e5e1-463f-9090-fcba7f33d8e6.png)
(Exception here)

System.InvalidCastException: 'Unable to cast object of type 'Files.Filesystem.StorageItems.SystemStorageFile' to type 'Windows.Storage.StorageFile'.'

Reply:
> Also, this won't work with remote storage (phone storage etc.)

ToStorageItem should work everywhere, including network and phone storage ;)